src/main/java/W2/Race.java

```diff
@@ -0,0 +1,119 @@
+package W2;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Race {
+    ArrayList<Car> cars=new ArrayList<>();
```

Comment:
```suggestion
    List<Car> cars=new ArrayList<>();
```
위처럼 작성하는게 좀더 일반적인 코드같습니다